bin/rspec

```diff
@@ -1,8 +1,29 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path('../spring', __FILE__)
-rescue LoadError => e
-  raise unless e.message.include?('spring')
+# frozen_string_literal: true
```

Comment:
I regenerated this binstub using `bundle binstub rspec-core --force` to get the non-springified version.